modules/pages/client/components/Faq.component.js

```diff
@@ -0,0 +1,346 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Board from '@/modules/core/client/components/Board.js';
+import { Trans, useTranslation } from 'react-i18next';
+import classnames from 'classnames';
+
+export default function Faq({ category, invitationsEnabled, children }) {
+  const { t } = useTranslation('pages');
+
+  return (
+    <>
+      <Board className="faq-header" names={['happyhippies', 'guitarcamper']}>
+        <div className="container">
+          <div className="row">
+            <div className="col-xs-12 text-center">
+              <h2 className="visible-xs-block">{t('FAQ')}</h2>
+              <h2 className="hidden-xs">{t('Frequently Asked Questions')}</h2>
+              {category === 'general' && (
+                <h3>{t('about the site & community')}</h3>
+              )}
+              {category === 'tribes' && <h3>{t('about tribes')}</h3>}
+              {category === 'foundation' && (
+                <h3>{t('about the foundation')}</h3>
+              )}
+              {category === 'technology' && <h3>{t('about technology')}</h3>}
+            </div>
+          </div>
+        </div>
+      </Board>
+
+      <section className="container">
+        <div className="row">
+          <div className="col-xs-12 col-sm-4 col-md-4">
+            <div
+              id="faq-sidebar"
+              className={classnames('sidebar', {
+                'sidebar-sticky': false,
+                // TODO implement waypoint and sticky sidebar feature by setting sidebar-sticky to: faq.waypoints.flags.on && faq.allowStickySidebar,
+              })}
+              // TODO zum-waypoint="faq.waypoints"
+              // TODO down="flags.on"
+              // TODO up="flags.off"
```

Comment:
You can remove the commented out stuff too. I think if you/we/someone decides sticky behaviour would be desirable, it would get re-implemented in a modern way.

On that, when doing a quick search for a react sticky library, I found this [1], which told me that CSS has a `position: sticky` option now, it's not usable in all browsers, but that would be OK for us.

The description of what it does on https://developer.mozilla.org/en-US/docs/Web/CSS/position hurt my brain:

> The element is positioned according to the normal flow of the document, and then offset relative to its nearest scrolling ancestor and containing block (nearest block-level ancestor), including table-related elements, based on the values of top, right, bottom, and left. 

We don't need to add it, but thought it was worth mentioning :) Some people like CSS puzzles...

[1] https://www.npmjs.com/package/react-sticky#css

Reply:
Ohh, I was just reading about sticky CSS another day and should've remembered to mention it here. Totally not necessary, but it'd be great if it would work. Ok for follow-ups too.

Reply:
Alright, I took that out and left a note about implementing 'position: sticky' in the future